libs/jose/Makefile

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (C) 2018 OpenWrt.org
```

Comment:
no, you don't have copyright assignment to openwrt.  Either drop it completely, or use your own name

Reply:
Thanks for review.